LinearAlgebraForCAP/examples/Example.g

```diff
@@ -141,4 +141,112 @@ right_side := PreCompose( [ i1, DualOnMorphisms( u ), u ] );;
+                cohom_ab
+              );
+#! <A morphism in Category of matrices over Q>
+ev_ab := EvaluationMorphism( a, b );
+#! <A morphism in Category of matrices over Q>
+coev_ab := CoevaluationMorphism( a, b );
+#! <A morphism in Category of matrices over Q>
+cocl_ev_ab := CoclosedEvaluationMorphism( a, b );
+#! <A morphism in Category of matrices over Q>
+cocl_ev_ba := CoclosedEvaluationMorphism( b, a );
+#! <A morphism in Category of matrices over Q>
+cocl_coev_ab := CoclosedCoevaluationMorphism( a, b );
+#! <A morphism in Category of matrices over Q>
+UnderlyingMatrix( ev_ab ) = TransposedMatrix( UnderlyingMatrix( cocl_ev_ba ) );
+#! true
+UnderlyingMatrix( coev_ab ) = TransposedMatrix( UnderlyingMatrix( cocl_coev_ab ) );
+#! true
+tensor_hom_adj_1_hom_ab := InternalHomToTensorProductAdjunctionMap( a, b, 1_hom_ab );
+#! <A morphism in Category of matrices over Q>
+cohom_tensor_adj_1_cohom_ab := InternalCoHomToTensorProductAdjunctionMap( a, b, 1_cohom_ab );
+#! <A morphism in Category of matrices over Q>
+tensor_hom_adj_1_ab := TensorProductToInternalHomAdjunctionMap( a, b, 1_ab );
+#! <A morphism in Category of matrices over Q>
+cohom_tensor_adj_1_ab := TensorProductToInternalCoHomAdjunctionMap( a, b, 1_ab );
+#! <A morphism in Category of matrices over Q>
+ev_ab = tensor_hom_adj_1_hom_ab;
+#! true
+cocl_ev_ab = cohom_tensor_adj_1_cohom_ab;
+#! true
+coev_ab = tensor_hom_adj_1_ab;
+#! true
+cocl_coev_ab = cohom_tensor_adj_1_ab;
+#! true
+c := VectorSpaceObject(2,Q);
+#! <A vector space object over Q of dimension 2>
+d := VectorSpaceObject(1,Q);
+#! <A vector space object over Q of dimension 1>
+pre_compose := MonoidalPreComposeMorphism( a, b, c );
+#! <A morphism in Category of matrices over Q>
+post_compose := MonoidalPostComposeMorphism( a, b, c );
+#! <A morphism in Category of matrices over Q>
+pre_cocompose := MonoidalPreCoComposeMorphism( c, b, a );
+#! <A morphism in Category of matrices over Q>
+post_cocompose := MonoidalPostCoComposeMorphism( c, b, a );
+#! <A morphism in Category of matrices over Q>
+UnderlyingMatrix( pre_compose ) = TransposedMatrix( UnderlyingMatrix( pre_cocompose ) );
+#! true
+UnderlyingMatrix( post_compose ) = TransposedMatrix( UnderlyingMatrix( post_cocompose ) );
+#! true
+tp_hom_comp := TensorProductInternalHomCompatibilityMorphism( [ a, b, c, d ] );
+#! <A morphism in Category of matrices over Q>
+cohom_tp_comp := InternalCoHomTensorProductCompatibilityMorphism( [ b, d, a, c ] );
+#! <A morphism in Category of matrices over Q>
+UnderlyingMatrix( tp_hom_comp ) = TransposedMatrix( UnderlyingMatrix( cohom_tp_comp ) );
+#! true
+lambda := LambdaIntroduction( alpha );
+#! <A morphism in Category of matrices over Q>
+lambda_elim := LambdaElimination( a, b, lambda );
+#! <A morphism in Category of matrices over Q>
+alpha = lambda_elim;
+#! true
+alpha_op := VectorSpaceMorphism( b, TransposedMatrix( UnderlyingMatrix( alpha ) ), a );
+#! <A morphism in Category of matrices over Q> 
```

Comment:
Please delete the trailing whitespace here.

Reply:
Done.